deploy/terraform-datacommons-website/modules/apikeys/variables.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type        = string
+  description = "GCP project id where the API keys will be created."
+}
+
+variable "website_domain" {
+  type        = string
+  description = "Domain name that you own that will be used for the Data Commons website."
```

Comment:
do all these definition have to happen in both modules and examples folder?

Reply:
variables in the examples are a subset of variables in the modules. Modules provide anything that can possibly configured and examples cherry-pick the default based on what kind of example it is creating. 

The duplication is kind of confusing but I'm not aware of any other method :(